packages/core/src/components/button/_common.scss

```diff
@@ -184,6 +188,10 @@ $non-default-state-selectors: ":last-child, :hover, :active, .pt-active, [class*
     background-image: none;
     color: $dark-button-color-disabled;
   }
+
+  .pt-button-spinner-container .pt-spinner-head {
```

Comment:
Indeed we don't have much choice for non-minimal intent buttons, the spinner needs to be white